sdk/cosmosdb/cosmos/src/common/constants.ts

```diff
@@ -181,9 +181,18 @@ export const Constants = {
 
   // Platform
   CurrentVersion: "2020-07-15",
+  /**
+   * @hidden
+   */
```

Comment:
What is this annotation doing?
I am wondering if these were not hidden earlier, hiding them now would be considered a breaking change.